scripts/generate_access_token.sh

```diff
@@ -25,11 +25,9 @@ set -euo pipefail
 # if a .env exists then source it
 if [ -f aad.env ]; then
   source aad.env
-else
-    echo "No aad.env file found. Run make deploy-ms-idp."
-    exit 1
 fi
 
-curl -X POST https://login.microsoftonline.com/$TenantId/oauth2/v2.0/token \
-	-H "Content-Type: application/x-www-form-urlencoded" \
-	-d "client_id=$ClientApplicationId&client_secret=$ClientSecret&scope=$ApiIdentifierUri/.default&grant_type=client_credentials"
+curl -X POST ${AadEndpoint:-http://localhost:3000/token} \
+    -H "Content-Type: application/x-www-form-urlencoded" \
+    -d "client_id=${ClientApplicationId:-}&client_secret=${ClientSecret:-}&scope=${ApiIdentifierUri:-}/.default&grant_type=client_credentials" \
```

Comment:
It probably doesnt matter for this PR but I presume in the B&A services, we will obtain token using managed identities not client secrets right?

Reply:
yes we agreed that. This will  generate access tokens for testing